packages/ui/src/components/carousel-item/carouselItem.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+  import { computed, getCurrentInstance, inject, onMounted, onUnmounted } from 'vue';
+  import { isDefined } from '@vueuse/core';
+  import type {
+    AcvCarouselItemProps,
+    AcvCarouselItemSlots
+  } from './carouselItem.ts';
+
+  import {
+    CAROUSEL_INJECTION_KEY
+  } from '@/components/carousel/carousel.ts';
+  import './carouselItem.css';
+
+  /**
+   * Short description of the CarouselItem component for the generated documentation...
+   *
+   * @component AcvCarouselItem
+   */
+  defineOptions({
+    name: 'AcvCarouselItem'
+  });
+
+  const props = defineProps<AcvCarouselItemProps>();
+
+  // defineEmits<AcvCarouselItemEvents>();
+  defineSlots<AcvCarouselItemSlots>();
+
+  const name = computed(() => props.name || getCurrentInstance()?.uid);
+
+  const carouselInjection = inject(CAROUSEL_INJECTION_KEY);
```

Comment:
can we remove all spaces between consts? it's first component that have it on each line